src/test/ui/parser/break-with-labeled-loop.rs

```diff
@@ -0,0 +1,16 @@
+// Regression test for issue #86948.
```

Comment:
This new test is technically a duplicate of the already existing one below. Maybe just add a comment to that one and add a check for the assignment expression instead of adding a new test.

Reply:
Fixed. Thanks for reviewing this!